robottelo/constants.py

```diff
@@ -1224,3 +1117,414 @@
     'Installable',
     '7/27/16'
 ]
+
+# To create custom role for bugzilla's
+# Append bugzilla id to dict, also see utils.get_perms_by_bz()
+PERMISSIONS_WITH_BZ = {
+    None: [
+        {'name': 'access_dashboard', 'bz': [1306359]},
+        {'name': 'access_settings', 'bz': [1306359]},
+        {'name': 'app_root', 'bz': []},
```

Comment:
@rplevka I prefer not having to deal with None values if possible, just like @san7ket did. More than that, all permissions are explicitly listed. But I agree empty lists all around the code are "ugly". So defaultdict can be used to avoiding definening a bz key where there is no bz id:

```python
>>> from collections import defaultdict
>>> dct=defaultdict(list)
>>> dct['bz']
[]
```

What you guys think?